src/modules/fw_pos_control_l1/fw_pos_control_l1_params.c

```diff
@@ -87,6 +87,18 @@ PARAM_DEFINE_FLOAT(FW_L1_DAMPING, 0.75f);
  */
 PARAM_DEFINE_FLOAT(FW_THR_CRUISE, 0.6f);
 
+/**
+ * Scale throttle by altitude AMSL
+ *
+ * This parameter enables throttle scaling based on barometric pressure
+ * When flying without airspeed sensor this will help to keep a constant performance
+ * over large altitude ranges.
+ *
+ * @boolean
+ * @group FW L1 Control
+ */
+PARAM_DEFINE_INT32(FW_THR_ALT_SC_EN, 0);
```

Comment:
Thus should be a float that multiplies by the pressure ratio to set the scale correction. A fixed ratio of 1 is not going to work for all propulsion types.